charts/core/src/Chart.stories.tsx

```diff
@@ -209,6 +209,14 @@ export default {
         disable: true,
       },
     },
+    groupId: {
+      description: 'Group ID for synced tooltips',
+      name: 'groupId',
+      table: {
+        category: 'Chart',
+        disable: true,
```

Comment:
curious: I don't see this in the controls panel. Does `disable: true` hide it?

Reply:
Yup, that's exactly it. If I don't add this it gives user the option to set the `groupId` which felt like it perhaps didn't demonstrate anything? Instead I just set them manually to the same thing in the `WithSameGroupIds` story so that they sync